manifest

```diff
@@ -11,13 +11,22 @@ export DOCUMENTATION_URL="https://chimeraos.org/about"
 export BUG_REPORT_URL="https://github.com/ChimeraOS/chimeraos/issues"
 
 export PACKAGES="\
-	gnome-desktop \
 	gnome-control-center \
 	gnome-session \
 	gnome-shell \
 	gnome-software \
 	gnome-terminal \
+	gnome-keyring \
```

Comment:
I am not certain what this is needed for.

Reply:
Password storage. Nothing more. It is helpful for any browser usage (although we don't provide any, flathub is at the ready). It can also help on locking ssh keys if needed.